src/sentry/static/sentry/app/views/organizationEventsV2/table.jsx

```diff
@@ -22,6 +24,14 @@ export default class Table extends React.Component {
       return <LoadingIndicator />;
     }
 
+    if (data && data.length === 0) {
```

Comment:
Can `data` be `null` or falsey?

Reply:
It *shouldn't* be if we're not hitting the loading condition above. Will probably need to add an error condition as well though.